### ISSUE TITLE
Cannot pass --kube-context to plugin

### ISSUE THREAD

**Issue by marckhouzam** [helm/helm-2to3#128]
When doing
```
helm 2to3 convert --kube-context lab myrelease
```
the `kube-context` seems to be ignored.
I looked a bit into the code and here is my impression of what is happening.

The `--kube-context` flag is being swallowed by helm and never reaches the plugin.  This is expected behaviour for helm global flags (see https://helm.sh/docs/topics/plugins/#a-note-on-flag-parsing).  

I think the plugin should instead read the `HELM_KUBECONTEXT` variable to figure out if the user has specified a context on the command-line.  This also implies the plugin should not have its own `--kube-context` flag but rely on helm's global one.

Another scenario that probably does not work right now is:
```
HELM_KUBECONTEXT=lab helm 2to3 convert myrelease
```

**Comment by hickeyma** [comment]
Thanks for catching this @marckhouzam. I tested using the binary only and not as a plugin as well.

I found the doc reference above as well. Its unfortunate that the plugin framework is consuming these flags in this way. 

I am not sure that I am a fan of the env variables as a solution. I wonder if using different flag names would be a better solution?

**Comment by marckhouzam** [comment]
> I am not sure that I am a fan of the env variables as a solution. I wonder if using different flag names would be a better solution?

The current helm plugin solution gives you both possibilities.  By switching the 2to3 plugin code to read the context from $HELM_KUBECONTEXT it will support both ways of specifying a context, from the variable, but more importantly from the flag:
```
$ helm 2to3 --kube-context lab convert myrelease
```

**Comment by hickeyma** [comment]
Fixed with PR #129
